examples/01_plotting/plot_prob_atlas.py

```diff
@@ -24,76 +24,94 @@
 # Load 4D probabilistic atlases
 from nilearn import datasets, plotting
 
-# Harvard Oxford Atlas
-harvard_oxford = datasets.fetch_atlas_harvard_oxford("cort-prob-2mm")
-harvard_oxford_sub = datasets.fetch_atlas_harvard_oxford("sub-prob-2mm")
+# Allen RSN networks
+allen = datasets.fetch_atlas_allen_2011()
 
-# Multi Subject Dictionary Learning Atlas
-msdl = datasets.fetch_atlas_msdl()
+# ICBM tissue probability
+icbm = datasets.fetch_icbm152_2009()
 
-# Smith ICA Atlas and Brain Maps 2009
-smith_rsn10 = datasets.fetch_atlas_smith_2009(resting=True, dimension=10)[
-    "map"
-]
-smith_rsn20 = datasets.fetch_atlas_smith_2009(resting=True, dimension=20)[
-    "map"
-]
-smith_rsn70 = datasets.fetch_atlas_smith_2009(resting=True, dimension=70)[
-    "map"
-]
-smith_bm10 = datasets.fetch_atlas_smith_2009(resting=False, dimension=10)[
-    "map"
-]
+# Smith ICA BrainMap 2009
 smith_bm20 = datasets.fetch_atlas_smith_2009(resting=False, dimension=20)[
     "map"
 ]
-smith_bm70 = datasets.fetch_atlas_smith_2009(resting=False, dimension=70)[
-    "map"
-]
 
-# ICBM tissue probability
-icbm = datasets.fetch_icbm152_2009()
-
-# Allen RSN networks
-allen = datasets.fetch_atlas_allen_2011()
+#########################################################################
+# Visualization
 
-# Pauli subcortical atlas
-subcortex = datasets.fetch_atlas_pauli_2017()
+# "contours" example
+plotting.plot_prob_atlas(allen.rsn28, title="Allen2011")
 
-# Dictionaries of Functional Modes (“DiFuMo”) atlas
-dim = 64
-res = 2
-difumo = datasets.fetch_atlas_difumo(
-    dimension=dim, resolution_mm=res, legacy_format=False
+# "continuous" example
+plotting.plot_prob_atlas(
+    (icbm["wm"], icbm["gm"], icbm["csf"]), title="ICBM tissues"
 )
 
-#########################################################################
-# Visualization
-atlas_types = {
-    "Harvard_Oxford": harvard_oxford.maps,
-    "Harvard_Oxford sub": harvard_oxford_sub.maps,
-    "MSDL": msdl.maps,
-    "Smith 2009 10 RSNs": smith_rsn10,
-    "Smith2009 20 RSNs": smith_rsn20,
-    "Smith2009 70 RSNs": smith_rsn70,
-    "Smith2009 20 Brainmap": smith_bm20,
-    "Smith2009 70 Brainmap": smith_bm70,
-    "ICBM tissues": (icbm["wm"], icbm["gm"], icbm["csf"]),
-    "Allen2011": allen.rsn28,
-    "Pauli2017 Subcortical Atlas": subcortex.maps,
-    f"DiFuMo dimension {dim} resolution {res}": difumo.maps,
-}
-
-for name, atlas in sorted(atlas_types.items()):
-    plotting.plot_prob_atlas(atlas, title=name)
-
-# An optional colorbar can be set
+# "filled_contours" example. An optional colorbar can be set.
 plotting.plot_prob_atlas(
-    smith_bm10,
-    title="Smith2009 10 Brainmap (with colorbar)",
+    smith_bm20,
+    title="Smith2009 20 Brainmap (with colorbar)",
     colorbar=True,
 )
-print("ready")
+
 plotting.show()
 
+#########################################################################
```

Comment:
Isn't there a more elegant way to do this ? Eg use triple quotes for the whole block so that people can easily copy-paste.

Reply:
Yes I think so. Let me see what renders best. I'll repost an updated link

Reply:
Updated docs: https://output.circle-artifacts.com/output/job/ff7c070a-3b19-409d-b7ea-6b1323283c17/artifacts/0/dev/auto_examples/01_plotting/plot_prob_atlas.html#sphx-glr-auto-examples-01-plotting-plot-prob-atlas-py